<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>606</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>916</width>
    <height>606</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>916</width>
    <height>606</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestione Visite</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>606</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>GestioneVisite.png</pixmap>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>88</y>
     <width>533</width>
     <height>434</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="button_inserisciVisita">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>INSERISCI VISITA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_eliminaVisita">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>160</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>ELIMINA VISITA</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="GestioneVisite.qrc"/>
 </resources>
 <connections/>
</ui>
